lib/BridgeCard.php

```diff
@@ -0,0 +1,259 @@
+<?php
+final class BridgeCard {
+
+	private static function buildFormatButtons($formats) {
+		$buttons = '';
+
+		foreach($formats as $name) {
+			$buttons .= '<button type="submit" name="format" value="'
+			. $name
+			. '">'
+			. $name
+			. '</button>'
+			. PHP_EOL;
+		}
+
+		return $buttons;
+	}
+
+	private static function getFormHeader($bridgeName, $isWarning = false) {
```

Comment:
I would probably rename isWarning to something clearer, like ```hasHTTPS```, or ```usesInsecureConnection```

Reply:
I'll go with 'isHttps' and change the conditions accordingly